crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -17,12 +17,19 @@ pub(crate) const DEFAULT_HOST: &str = "127.0.0.1";
 
 pub(crate) struct Server {
     pub server: hyper::Server<hyper::server::conn::AddrIncoming, IntoMakeService<Router>>,
+    #[cfg(test)]
+    pub schema: async_graphql::Schema<Query, EmptyMutation, EmptySubscription>,
```

Comment:
Instead of doing this, can you add a private function to the builder to just return the schema?